Symfony/CS/Tokenizer/Token.php

```diff
@@ -382,8 +382,7 @@ public function isStructureAlternativeEnd()
     /**
      * Check if token is a whitespace.
      *
-     * @param array  $opts                array of extra options
-     * @param string $opts['whitespaces'] string determining whitespaces chars, default is " \t\n\r\0\x0B"
+     * @param array $opts array of extra options, $opts['whitespaces'] string determining whitespaces chars, default is " \t\n\r\0\x0B"
```

Comment:
Will do, although I still think hinting a second `@param` is not good practice if there isn't a second param in the method signature.